soccer/gameplay/plays/offense/cross_kick.py

```diff
@@ -0,0 +1,272 @@
+
+    
+    def on_enter_transitioning(self): 
+        self.remove_all_subbehaviors()
+        
+        point_flank = robocup.Point(constants.Field.Width * 0.4, constants.Field.Length *.78)
+        #move_flank = skills.move.Move(point_flank)
+        pass_to_flank = tactics.coordinated_pass.CoordinatedPass(point_flank)
+        #capture = skills.capture.Capture()
+
+        self.add_subbehavior(pass_to_flank, 'pass to flank', required=True)
+        #self.add_subbehavior(move_flank, 'move to flank', required=True)
+        #self.add_subbehavior(capture, 'capture', required=True) 
+
+    def on_exit_transitioning(self):
+        self.remove_all_subbehaviors()
+
+    def on_enter_transitioningLeft(self): 
+        self.remove_all_subbehaviors()
+        
+        point_flank = robocup.Point(constants.Field.Width * -0.4, constants.Field.Length *.78)
+        #move_flank = skills.move.Move(point_flank)
+        pass_to_flank = tactics.coordinated_pass.CoordinatedPass(point_flank)
+        #capture = skills.capture.Capture()
+
+        self.add_subbehavior(pass_to_flank, 'pass to flank', required=True)
+        #self.add_subbehavior(move_flank, 'move to flank', required=True)
+        #self.add_subbehavior(capture, 'capture', required=True) 
+
+    def on_exit_transitioningLeft(self):
+        self.remove_all_subbehaviors()
+   
+    def on_enter_transitioningDeep(self): 
+        self.remove_all_subbehaviors()
+        
+        point_flank = robocup.Point(constants.Field.Width * 0.4, constants.Field.Length *.5)
+        #move_flank = skills.move.Move(point_flank)
+        pass_to_flank = tactics.coordinated_pass.CoordinatedPass(point_flank)
+        #capture = skills.capture.Capture()
+
+        self.add_subbehavior(pass_to_flank, 'pass to flank', required=True)
+        #self.add_subbehavior(move_flank, 'move to flank', required=True)
+        #self.add_subbehavior(capture, 'capture', required=True) 
+
+    def on_exit_transitioningDeep(self):
+        self.remove_all_subbehaviors()
+
+    def on_enter_transitioningDL(self): 
+        self.remove_all_subbehaviors()
+        
+        point_flank = robocup.Point(constants.Field.Width * -0.4, constants.Field.Length *.5)
+        #move_flank = skills.move.Move(point_flank)
+        pass_to_flank = tactics.coordinated_pass.CoordinatedPass(point_flank)
+        #capture = skills.capture.Capture()
+
+        self.add_subbehavior(pass_to_flank, 'pass to flank', required=True)
+        #self.add_subbehavior(move_flank, 'move to flank', required=True)
+        #self.add_subbehavior(capture, 'capture', required=True) 
+
+    def on_exit_transitioningDL(self):
+        self.remove_all_subbehaviors()
+    
+    def on_enter_crossing(self):
+        self.remove_all_subbehaviors()
+        
+        #capture = skills.capture.Capture()
+        #point_near_box = robocup.Point(-.4,9*.83)
+        point_in_box = robocup.Point(constants.Field.Width * -0.1, constants.Field.Length * .875)
```

Comment:
Are we ok to having a hard coded points in general? What do we do if an enemy robot is in that position? Should we use any other kick location selector?